Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,29 @@
+# Release Notes for MultiplayerSample
+
+## O3DE 2305 Release
+
+![2305 Screenshot](Media/2305_screenshot.png)
+
+2305 is the base version of the game. It is expected to be functional and support up to 10 players. 
+
+### Testing Notes
+
+| Platform         | In Editor    | Profile PAKs | Release + PAKs |
+|------------------|--------------|--------------|----------------|
+| Windows          | yes - stable | yes - stable | yes - unstable |
```

Comment:
It's been a few months since we tested Release + PAKs, i'd suggest calling that "untested"